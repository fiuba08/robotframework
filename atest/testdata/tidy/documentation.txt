*** Settings ***
Documentation    Hello
...    - list
...    - <world>

*** Test Cases ***
Multiline
    [ DocuMentation ]    Hillo
    ...  | table |
    ...  | world |
Multiline with manual line separators
    [Document]    Hillo\non\n hyvää!
One line
    [Documentation]    One line
Empty
    [Documentation]
None
    No Operation

*** Keywords ***
Keyword doc
    [Documentation]  Multi
    ...  line\nstuff
