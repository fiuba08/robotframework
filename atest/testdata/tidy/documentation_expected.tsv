*Settings*							
Documentation	Hello						
...	- list						
...	- <world>						
							
*Test Cases*							
Multiline	[Documentation]	Hillo					
	...	| table |					
	...	| world |					
							
Multiline with manual line separators	[Documentation]	Hillo					
	...	on					
	...	hyvää!					
							
One line	[Documentation]	One line					
							
Empty							
							
None	No Operation						
							
*Keywords*							
Keyword doc	[Documentation]	Multi					
	...	line					
	...	stuff					
